src/main/java/it/pagopa/ecommerce/payment/methods/application/v1/PaymentMethodService.java

```diff
@@ -356,9 +357,12 @@ public Mono<CreateSessionResponseDto> createSessionForPaymentMethod(
                     SessionPaymentMethod sessionPaymentMethod = SessionPaymentMethod
                             .fromValue(paymentMethod.serviceName);
                     URI returnUrlBasePath = sessionUrlConfig.basePath();
-                    URI resultUrl = returnUrlBasePath.resolve(sessionUrlConfig.outcomeSuffix());
-                    URI merchantUrl = returnUrlBasePath;
-                    URI cancelUrl = returnUrlBasePath.resolve(sessionUrlConfig.cancelSuffix());
+                    URI resultUrl = UriComponentsBuilder
```

Comment:
please add assertions to tests to check that `t` parameter is effectively present into url

Reply:
Done here 08965f594a38d5832f613ab2a4ba56e7dd39e56f